example-project/mycmsproject/forms.py

```diff
@@ -8,3 +8,4 @@
 class ContentForm(forms.ModelForm):
     class Meta:
         model = Content
+        exclude = ()
```

Comment:
The recommended way to do this appears to be to set `fields = '__all__'`:
https://docs.djangoproject.com/en/1.8/topics/forms/modelforms/#selecting-the-fields-to-use